src/index.html

```diff
@@ -1,14 +1,50 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="/home" class="logo__img"></a>
+      </div>
+      <nav class="navbar">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a href="/apple" class="nav__link-link is-active">Apple</a>
+          </li>
```

Comment:
You name block `navbar` but child elements you named as `nav__child`.
`nav__link-link` - It is very bad to repeat names, especially in nested elements. You can name `li` element as `navbar__item`.


```suggestion
        <ul class="navbar__list">
          <li class="navbar__item">
            <a href="/apple" class="navbar__link is-active">Apple</a>
          </li>
```